doc/src/bond_comm_epoxy_wire.md

```diff
@@ -205,8 +207,12 @@ to re-handshake.
 The Bond Epoxy wire format has no provisions for communications security.
 
 Implementations of the Epoxy transport are encouraged to offer both plain
-TCP and TLS connections. Configuration of the TLS parameters should be
-deferred to the application.
+TCP and TLS connections. TLS 1.2 or greater must be used. Configuration of
+the other TLS parameters (e.g., allowed cypher suites) should be deferred to
```

Comment:
Cipher? Cypher is a character from the Matrix.
(This is apparently an alternate spelling, so you can keep it if you really like it. :) )